fixtures/complex.php.sem.uast

```diff
@@ -30,7 +30,7 @@
                Block: true,
                Prefix: "\n * ",
                Suffix: "\n ",
-               Tab: "",
+               Tab: " * ",
```

Comment:
This looks weird. If this is set, the comments should have changed as well. May be a bug.